src/globals/scss/core/_prose-scope.scss

```diff
@@ -48,5 +48,10 @@
     a {
       @extend .govuk-link;
     }
+
+    hr {
```

Comment:
Should this use govuk-section-break__visible ?

Reply:
Can we also add an example to the prose page to show this in use?

Reply:
Updated